go

```diff
@@ -4,12 +4,26 @@ require 'English'
 
 Dir.chdir File.dirname(__FILE__)
 
+def try_command_and_restart(command)
+  exit $CHILD_STATUS.exitstatus unless system command
+  env = {}.merge(ENV)
+  env.delete('RUBYOPT')
+  exec(env, RbConfig.ruby, *[$PROGRAM_NAME].concat(ARGV))
```

Comment:
Updated both the `GoScript::Template` class to emit some comments, and added the comments to this `./go` script instance.